script/webpack-analyze

```diff
@@ -9,5 +9,5 @@ WEBPACK_DIR="$FOREMAN_PATH/public/webpack"
 REPORT="${WEBPACK_DIR}/report.html"
 
 mkdir -p $WEBPACK_DIR
-$WEBPACK --profile --json > $WEBPACK_DIR/stats.json && \
+$WEBPACK --profile --json > $WEBPACK_DIR/stats.json
```

Comment:
why? this ensures that if webpack json output is not correct it will not run the analyzer.

Reply:
My bad, leftover from some debugging session (it prevented me from seeing some logs).